src/components/RadioButton/RadioButton.jsx

```diff
@@ -48,27 +52,42 @@ const RadioButton = forwardRef(
     }, [checked, defaultChecked]);
 
     return (
-      <label className={cx(baseClassName, overrideClassName, { disabled })}>
-        <span className={`${baseClassName}__radio-input-container`}>
+      <label
+        className={cx(styles.radioButton, CSS_BASE_CLASS, overrideClassName, {
+          [styles.disabled]: disabled,
+          ["disabled"]: disabled
+        })}
+      >
+        <span className={cx(styles.inputContainer, "monday-style-radio-button-component__radio-input-container")}>
           <input
-            className={`${baseClassName}__radio-input-container__radio-input`}
+            className={cx(styles.radioInput, "monday-style-radio-button-component__radio-input-container__radio-input")}
             type="radio"
             value={value}
             name={name}
             disabled={disabled}
             {...checkedProps}
             onChange={onSelect}
             ref={mergedRef}
+            id={id}
+            data-testid={dataTestId || getTestId(ELEMENT_TYPES.RADIO_BUTTON, id)}
           />
           <span
-            className={cx(`${baseClassName}__radio-input-container__radio-control`, {
-              [`${baseClassName}__radio-input-container__radio-control--label-animation`]: !noLabelAnimation
-            })}
+            className={cx(
+              styles.radioControl,
+              "monday-style-radio-button-component__radio-input-container__radio-control",
+              {
+                [styles.radioControlLabelAnimation]: !noLabelAnimation,
+                ["monday-style-radio-button-component__radio-input-container__radio-control--label-animation"]:
+                  !noLabelAnimation
+              }
+            )}
           />
         </span>
-        {text && <span className={`${baseClassName}__radio-label`}>{text}</span>}
+        {text && (
+          <span className={cx(styles.radioLabel, "monday-style-radio-button-component__radio-label")}>{text}</span>
+        )}
         {children && (
-          <Clickable className="radio-children-wrapper" onClick={onChildClick} tabIndex={childrenTabIndex}>
+          <Clickable className={cx("radio-children-wrapper")} onClick={onChildClick} tabIndex={childrenTabIndex}>
```

Comment:
@SergeyRoyt, it looks like this classname is either non existing or wasn't transformed?

Reply:
Yes, such class just doesn't exist